src/main/scala/akka/minion/HttpServer.scala

```diff
@@ -126,6 +126,12 @@ object Template {
           rel := "stylesheet",
           href := "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css"
         ),
+        script(
+          src := "https://code.jquery.com/jquery-3.3.1.min.js"
+        ),
+        script(
+          src := "https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"
+        ),
```

Comment:
Is `jquery-ui` needed or would `jquery` alone is enough?

Reply:
@2m it shows same error with `jquey` only.

Reply:
Are you sure? I do not see a dependency to jquery-ui in the bootstraps bower configuration: https://github.com/twbs/bootstrap/blob/0b9c4a4007c44201dce9a6cc1a38407005c26c86/bower.json#L32

Reply:
@2m oh, sorry for confusion. You're right. It was maybe caused because cache was left. I updated it.

Reply:
Yea. Cache could explain it. Thank you very much for looking into it again!